### DISCUSSION TITLE
Telegram stickers pack to Signal, Mole, GreenLezard etc

### DISCUSSION THREAD

**Discussion by Ivanacz** [signalstickers/signalstickers#837] [Stickers requests]
I was trying to get these into signal the last couple of days but I can't get the file size under 300kb of the apng so I would be able to create sticker pack in signal app.

I'm not even familiar with python, so I don't know how to work with the script (Trust me, I tried) I was hoping someone could help me and convert my favorite animated stickers into signal and keep them still animated or us much is possible. I would highly appreciate that. I was hoping I can figure it out, but I have to give up :(

https://tgram.ru/stickers/GreenLezard
https://tgram.ru/stickers/TheLittleMole
https://tgram.ru/stickers/ShadowKitty
https://tgram.ru/stickers/ShaunTheSheep
https://tgram.ru/stickers/RickAndMorty

**Comment by MotherLilith** [comment]
Hello, 
I have also been trying to convert my favourite animated Telegram stickers to Signal but I just have to give up since I am no tech wiz. I absolutely adore Boy's Club stickers from Telegram (https://tlgrm.eu/stickers/BoysClub) and
I would appreciate it if some kind soul could convert these to animated Signal stickers. Thank you already in advance! 😌🙏🏻❤️

**Comment by dreua** [comment]
If I may also add my requests here:
- [x] Proper Harold Pack V.2 https://www.stickerstelegram.com/characters/proper-harold-pack-v-2/ - [Done, thanks @ingria](https://signal.art/addstickers/#pack_id=675d520880f029cd0d18ac03f587c2f1&pack_key=5902ba80f231234e90199bc1d5f655ed2c3f310f412f841ff6258e2d8750968d)
- [x] https://t.me/addstickers/Animalfeelings - [Done, thanks @ingria](https://signal.art/addstickers/#pack_id=9472974b55fcfa1c3881fd259116a22c&pack_key=ce946afd78ba27d9c728bc579797d0dc568c305fa337fa5cc48a37dbd962525a)
- [ ] https://t.me/addstickers/FluffyLoafer

Would be great, thanks!

**Reply by ingria** [reply]
https://signal.art/addstickers/#pack_id=675d520880f029cd0d18ac03f587c2f1&pack_key=5902ba80f231234e90199bc1d5f655ed2c3f310f412f841ff6258e2d8750968d

https://signal.art/addstickers/#pack_id=9472974b55fcfa1c3881fd259116a22c&pack_key=ce946afd78ba27d9c728bc579797d0dc568c305fa337fa5cc48a37dbd962525a

**Comment by coemgen1992** [comment]
I may add this sticker set to the request list, thanks!

https://telegramhub.net/cloudia-sticker-pack/
